peer/brontide.go

```diff
@@ -4456,17 +4462,32 @@ func (p *Brontide) finalizeChanClosure(chanCloser *chancloser.ChanCloser) {
 	// If this is a locally requested shutdown, update the caller with a
 	// new event detailing the current pending state of this request.
 	if closeReq != nil {
+		// TODO(roasbeef): don't actually need this?
 		closeReq.Updates <- &PendingUpdate{
-			Txid: closingTxid[:],
+			Txid:           closingTxid[:],
+			IsLocalCloseTx: fn.Some(true),
```

Comment:
nit: looks like it's unrelated and it's from the RBF coop close PR😂

Reply:
Hmm, yeah I was testing this rebased on top of that branch to exercise the expected behavior. Removing for now, will revisit as there may have been some undocumented assumption here.